### PR TITLE
RavenDB-16185 Add global full change vector for replication handshake

### PR DESCRIPTION
**Pull request by efratshenhar**

### CODE REVIEW

File: src/Raven.Server/Documents/Replication/ReplicationLoader.cs [hunk 1]
```diff
@@ -367,7 +367,7 @@ private ReplicationLatestEtagRequest IncomingInitialHandshake(TcpConnectionOptio
                 using (documentsOperationContext.OpenReadTransaction())
                 using (configurationContext.OpenReadTransaction())
                 {
-                    var changeVector = DocumentsStorage.GetDatabaseChangeVector(documentsOperationContext);
+                    var changeVector = DocumentsStorage.GetFullDatabaseChangeVector(documentsOperationContext);
```
Comment (ayende) [inline src/Raven.Server/Documents/Replication/ReplicationLoader.cs:370]: Shouldn't we default to the database change vector if empty?

Comment (efratshenhar) [reply src/Raven.Server/Documents/Replication/ReplicationLoader.cs:370]: if full CV is empty so does regular database CV

File: src/Raven.Server/Documents/DocumentsStorage.cs [hunk 2]
```diff
@@ -515,6 +519,31 @@ public void SetDatabaseChangeVector(DocumentsOperationContext context, string ch
...
         }
 
+        public static string GetFullDatabaseChangeVector(DocumentsOperationContext context)
+        {
+            var tx = context.Transaction.InnerTransaction;
+            var tree = tx.ReadTree(GlobalTreeSlice);
+            var val = tree.Read(GlobalFullChangeVectorSlice);
+            if (val == null)
+            {
+                return string.Empty;
```
Comment (karmeli87) [inline src/Raven.Server/Documents/DocumentsStorage.cs]: Let's return the normal database change vector here.
Upgrading an existing server will have null as full database change vector..

**Review by karmeli87 — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Documents/DocumentsStorage.cs (hunk 2 above)
Comment (efratshenhar) [reply src/Raven.Server/Documents/DocumentsStorage.cs]: ok

**Review by karmeli87 — APPROVED** [review]

**Review by ayende — APPROVED** [review]

**Pull request merged**